### PR TITLE
Introduced support for NoSQLClient as MongoDB management tool

### PR DESCRIPTION
**Pull request by chicco785**
* provided reference to docker compose code lines
* commented out the ports section (it’s needed only to expose mongo outside on the public network)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ Few hints, to customize the [`docker-compose.yml`](docker-compose.yml) orchestra
 - [x] The script is able to add and remove nodes dynamically according to the evolution of the swarm cluster
 - [x] Given the about restart on failure is recommended as policy, this ensure that the scripts restart when it exit -1 and when the node where it is running is removed / drained (you need more than one master node!)
 - [x] The repository includes a basic set of Travis CI tests that tests the script behavior against basic conditions in a single swarm node and without data persistence (initialization, scale up, scale down)
+- [x] The recipe include [Nosqlclient](https://www.nosqlclient.com) as mongodb management tool.
```
Comment (taliaga) [inline README.md]: typo: includes

File: README.md [hunk 2]
```diff
@@ -58,19 +58,19 @@ You can configure the following environment variables for deploying your stack u
 
 Few hints, to customize the [`docker-compose.yml`](docker-compose.yml) orchestration according to your needs:
 
-* To use data persistence (which we recommend in production settings), the *Mongo* service needs to be deployed in **global mode**. This is to avoid that more than one instance is deployed on the same node and that different instances concurrently access the same MongoDB data space on the filesystem.
+* To use data persistence (which we recommend in production settings), the *mongo* service needs to be deployed in **global mode** (see [`docker-compose.yml` line 5](docker-compose.yml#L5)). This is to avoid that more than one instance is deployed on the same node and that different instances concurrently access the same MongoDB data space on the filesystem.
```
Comment (taliaga) [inline README.md:61]: IMO lines numbers are quite fragile here, they'll become outdated quite soon and will be more a confusion than help.

**Review by taliaga — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (chicco785) [reply README.md:61]: of course these has to be updated in case. let's see.

File: README.md (hunk 1 above)
Comment (chicco785) [reply README.md]: thx, fixed!

**Pull request merged**